wp-tevko-responsive-images.php

```diff
@@ -297,31 +297,130 @@ function tevkori_get_src_sizes( $id, $size = 'thumbnail' ) {
 }
 
 /**
- * Filter for extending image tag to include srcset attribute.
+ * Filters images in post content to add `srcset` and `sizes`.
  *
- * @see images_send_to_editor
- * @return string HTML for image.
+ * @since 3.0
+ *
+ * @param string $content The raw post content to be filtered.
+ * @return string Converted content with `srcset` and `sizes` added to images.
+ */
+function tevkori_filter_content_images( $content ) {
+	// Only match images in our uploads directory.
+	$uploads_dir = wp_upload_dir();
+	$path_to_upload_dir = $uploads_dir['baseurl'];
```

Comment:
I think we should add this here...

```
$paths_to_upload_dir = apply_filters( 'tevkori_paths_to_upload_dir', array( $path_to_upload_dir ) );

foreach ( $paths_to_upload_dir as $path_to_upload_dir ) {
	$content = preg_replace_callback( '|<img ([^>]+' . $path_to_upload_dir . '[^>]+)[\s?][\/?]>|i', '_tevkori_filter_content_images_callback', $content );
}
```

This will allow plugins like [WP Offload S3](https://wordpress.org/plugins/amazon-s3-and-cloudfront/) to define additional paths to match.